### PR TITLE
[#113] Add a CLI option to print the version

### PR DESCRIPTION
**Pull request by fpavageau**
I'll repeat the operation for `master_3.x` once this has been closed.

### CODE REVIEW

**Comment by fpavageau** [comment]
This PR addresses #113.

File: liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java [hunk 1]
```diff
@@ -121,6 +139,21 @@ public static void main(String[] args) {
         );
     }
 
+    private static void printVersion() {
+        InputStream propsIs = LiquigraphCli.class.getResourceAsStream("/liquigraph-cli.properties");
```
Comment (fbiville) [inline liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java]: should be (auto-)closed, right?

File: liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java [hunk 2]
```diff
@@ -121,6 +139,21 @@ public static void main(String[] args) {
...
+        String version = null;
+        if (propsIs != null) {
+            Properties props = new Properties();
+            try {
+                props.load(propsIs);
+            } catch (IOException e) {
+                LOGGER.error("An exception occurred while loading the properties", e);
+            }
+            version = props.getProperty("liquigraph.version");
```
Comment (fbiville) [inline liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java]: Optional: I'd move the property extraction to another method

File: liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java (hunk 1 above)
Comment (fpavageau) [inline liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java]: Closing? What's that?

And I just added try-with-resources for connections in another branch...

File: liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java (hunk 2 above)
Comment (fpavageau) [inline liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java]: I hesitated to do it while implementing, as it seemed a bit overkill in this case: the only thing remaining if I move the property extraction is `System.out.println()`. It's no big deal anyway, so I'll do it.

**Comment by fbiville** [comment]
Merged and cherry-picked on `master_3.x` ;-)
Works well in both branches 👍

**Pull request merged**